eval.ml

```diff
@@ -8,7 +8,8 @@ type trace = value list
 type environment = binding Env.t
 type heap = heap_value Heap.t
 
-type status = Running | Stopped
+type status = Running | Stopped of value option
+type continuation = variable * environment * instruction_stream * pc
```

Comment:
it is either `(function_name, version_name, pc)` or `(instructions, pc)`.
I chose the second because currently the configuration does not keep track of the current function and version name.
But maybe for the long we should switch (eg. its better for printing backtraces).